<?xml version="1.0"?>
<launch>


  <include file="$(find setup)/launch/p3at_spawn.launch">
    <arg name="robotname" value="robot1"/>
    <arg name="initialy" value="47"/>
    <arg name="initialx" value="17"/>
  </include>

      <include file="$(find setup)/launch/p3at_spawn.launch">
        <arg name="robotname" value="robot0"/>
        <arg name="initialy" value="47"/>
        <arg name="initialx" value="3"/>
      </include>

            <include file="$(find setup)/launch/p3at_spawn.launch">
              <arg name="robotname" value="robot2"/>
              <arg name="initialy" value="44"/>
              <arg name="initialx" value="8"/>
            </include>


                  <include file="$(find setup)/launch/p3at_spawn.launch">
                    <arg name="robotname" value="robot3"/>
                    <arg name="initialy" value="44"/>
                    <arg name="initialx" value="12"/>
                  </include>




</launch>
